.github/workflows/velox_be.yml

```diff
@@ -394,6 +375,7 @@ jobs:
         run: |
           docker exec -i static-build-test-$GITHUB_RUN_ID bash -c '
           source /env.sh && \
+          sudo yum -y install patch && \
```

Comment:
move this to velox dep?

Reply:
I find `setup_linux` is called only for vcpkg compilation, so it does not work to add this command in setup-centos7.sh.
https://github.com/oap-project/gluten/blob/main/ep/build-velox/src/build_velox.sh#L76-L78